### PR TITLE
[PT-BR] Update various articles

### PR DESCRIPTION
**Pull request by Flantsune**
Updated both wiki entries in pt-br to reflect an accurate translation to Portuguese Brazilian of the current English versions on the live wiki. Now both currently up-to-date.

Updated a link in [BR] Rules about the wikipedia NSFW to link to a Brazilian Portuguese one, the article explains the very same idea as the English one.

Changes to [BR] Mascots look a tad weird in the commit because it still used the old layout for [EN] Mascots.

### CODE REVIEW

File: wiki/Mascots/pt-br.md [hunk 1]
```diff
@@ -1,229 +1,99 @@
...
-#### Altura
-?
+> Ele é o líder sangue quente da torcida. Ele tem um bom coração e ajuda os outros ao seu redor que estão em apuros através de se levantar e lutar por eles!
 
-#### Aparece em
-- Catch the Beat (como chibi-Ryuuta) [Substituído por Yuzu]
-- osu!webpage (chibi-Ryuuta como mascote) [Retirado]
-- [Skin](https://osu.ppy.sh/community/forums/topics/190357/)
-- Yandere Simulator (como um NPC especial)
+一本木龍太 (Ryūta Ippongi) era o fruta-chibi catcher do osu!catch que se juntou em 2008, mas foi substituído por [Yuzu](#-yuzu) em 2014. Ele foi criado pela [iNiS Corporation](https://en.wikipedia.org/wiki/INiS) e já fez parte do antigo website. Ele também apareceu em [Yandere Simulator](https://yanderesimulator.com) como um NPC.
```
Comment (Kuron-kun) [inline wiki/Mascots/pt-br.md]: substituir o último ele por ``Yuzu``? pra não repetir tanto a palavra "ele"

File: wiki/Mascots/pt-br.md [hunk 2]
```diff
@@ -1,229 +1,99 @@
...
-- osu!webpage (chibi-J como mascote) [Retirado]
-- [Skin](https://osu.ppy.sh/community/forums/topics/190357/)
-
-#### Criador
-iNiS Corporation
+![Don](img/Don.png "Don")
 
-#### Sexo
-Masculino
+> Don é o protagonista da série [Taiko no Tatsujin](https://en.wikipedia.org/wiki/Taiko_no_Tatsujin). Ele é um tambor de taiko com uma borda branca e quatro pernas, uma cara vermelha (que aparenta ser moly orange), e um corpo azul claro. O sonho de Don é compartilhar a beleza de Taiko para o mundo. Três anos se passaram desde que ele se mudou para a Casa Wada, e ele se tornou popular pela cidade. Ele tem um apetite monstruoso e as vezes tem compras caras na Casa Wada, que possa se assumir o pior. Ele geralmente termina suas frases com "Ta-don", significando o som de um tambor "Ba-dum" em Japonês.
```
Comment (Kuron-kun) [inline wiki/Mascots/pt-br.md]: ``[...] Casa Wada, e ele se tornou popular pela cidade`` > ``e ele acabou tornando-se popular pela cidade``? parece mais bonito visualmente, idk

File: wiki/Mascots/pt-br.md [hunk 3]
```diff
@@ -1,229 +1,99 @@
...
 
-#### Aparece em
-- Taiko (Skin)
-{Skin oficial para as [Opções do gatilho](/wiki/Options#Skin)}
+![Chirou](img/Chirou.png "Chirou")
 
-#### Criador
-- Yukiko Yokoo (横尾有希子) [Arte]
-- Narahashi Miki (楢橋 美紀) [Dublador]
+Projetada por [pyun](https://osu.ppy.sh/users/981534), Chirou era uma das participantes na competição de design de mascotes para osu!taiko. Ela é bem rigorosa e exigente, uma perfeccionista, e não gosta de errar—especialmente em suas beats no osu!taiko. Mas deixando sua perspectiva forte de lado, se você chegar em seu ponto fraco, ela pode ser bem fofa. Ela tem 14 anos de idade, nascida em 25 de Outubro e tendo seu tipo sanguíneo AB. Ela também mede 150 centímetros e pesa 45 quilogramas. Seus hobbies incluem praticar batucar, se aquecer em sua capa, e coletar gemas ou pedras. Chirou também apareceu em vários envios de fanarts de usuários.
```
Comment (Kuron-kun) [inline wiki/Mascots/pt-br.md]: mudar hífen por vírgula? acho que foi typo

File: wiki/Mascots/pt-br.md [hunk 4]
```diff
@@ -1,229 +1,99 @@
...
-Masculino
+### Taikonator
 
-#### Data de adição
-Maio de 2008
+![Taikonator](img/Taikonator.png "Taikonator")
 
-#### Peso
-100+ kg
+Taikonator, também conhecido como Taikonator 3000, era um dos participantes na competição de design de mascotes para osu!taiko. Ele ganhou certa popularidade como uma piada internapor razões desconhecidas. Suas origens continuam sendo um mistério; apesar disso, ele é bem mais único do que outros designs de mascote para osu!taiko e também apareceu bastante em vários envios de fanarts de usuários.
```
Comment (Kuron-kun) [inline wiki/Mascots/pt-br.md]: ``interna por``

**Comment by Flantsune** [comment]
I was supposed to create a new branch before trying to create a PR to not merge on this one, my sincere apologies.
The new commits come with more articles translated to PT-BR and should have a description on their own.

**Pull request closed without merging**